client/ayon_core/tools/publisher/control.py

```diff
@@ -1369,6 +1382,7 @@ def event_system(self):
             "publish.is_running.changed" - Attr 'publish_is_running' changed.
             "publish.has_crashed.changed" - Attr 'publish_has_crashed' changed.
             "publish.publish_error.changed" - Attr 'publish_error'
+            publish.publish_error_label.changed  - Attr 'publish_error_label'
```

Comment:
To match the others in the docstring.
```suggestion
            "publish.publish_error_label.changed"  - Attr 'publish_error_label'
```